front/app/components/HookForm/SlugInput/index.tsx

```diff
@@ -1,34 +1,46 @@
 import React from 'react';
 import messages from './messages';
 import { FormattedMessage, useIntl } from 'utils/cl-intl';
-import { Text } from '@citizenlab/cl2-component-library';
+import {
+  Text,
+  Label,
+  IconTooltip,
+  Box,
+} from '@citizenlab/cl2-component-library';
 import Warning from 'components/UI/Warning';
 import Input from 'components/HookForm/Input';
 
 interface Props {
   showWarningMessage: boolean;
-  previewUrl: string;
-  slug: string;
+  previewUrl: string | null;
+  slug?: string;
 }
 
 const SlugInput = ({ previewUrl, showWarningMessage, slug }: Props) => {
   const { formatMessage } = useIntl();
 
   return (
     <>
-      <Input
-        label={formatMessage(messages.urlSlugLabel)}
-        labelTooltipText={formatMessage(messages.slugTooltip)}
-        type="text"
-        name="slug"
-        value={slug}
-      />
-      <Text mb={showWarningMessage ? '16px' : '0'}>
-        <i>
-          <FormattedMessage {...messages.resultingURL} />
-        </i>
-        : {previewUrl}
-      </Text>
+      <Box display="flex" alignItems="center">
+        <Box mr="5px">
+          <Label htmlFor="slug-input">
+            {formatMessage(messages.urlSlugLabel)}
+          </Label>
+        </Box>
+        {/* Margin bottom is needed because of Label's default margin (which we need to remove) */}
+        <Box mb="10px">
+          <IconTooltip content={formatMessage(messages.slugTooltip)} />
+        </Box>
+      </Box>
```

Comment:
I needed to have a separate IconTooltip for RTL to work. It couldn't select the input because of the IconTooltip being in there (if I used the one built into Input). I'll make sure to log this.